tests/unit/disco/sagas/testDiscovery.js

```diff
@@ -0,0 +1,104 @@
+import SagaTester from 'redux-saga-tester';
+
+import { loadEntities } from 'core/actions';
+import { ErrorHandler } from 'core/errorHandler';
+import addonsReducer from 'core/reducers/addons';
+import apiReducer from 'core/reducers/api';
+import * as api from 'disco/api';
+import { getDiscoResults, loadDiscoResults } from 'disco/actions';
+import discoResultsReducer from 'disco/reducers/discoResults';
+import discoverySaga from 'disco/sagas/discovery';
+import { dispatchSignInActions } from 'tests/unit/amo/helpers';
+import {
+  createFetchDiscoveryResult, fakeDiscoAddon,
+} from 'tests/unit/disco/helpers';
+
+
+describe('disco/sagas/discovery', () => {
```

Comment:
I've noticed this is a pattern you've been using naming tests recently. While I think it's okay (it's readable and gives some insight into what the test is describing and what its context is), it's quite fragile as if a test moves (but maybe otherwise stayed the same) the description would need updating, compared to just calling it `Discovery Saga`. Jest does a great job of telling us which files are being run and where any errors are found as well, so I'd say using paths as test descriptors isn't a great habit to get into.

Let's say you had an error in the first test–writing "Discovery Saga" and getting an error here would result in Jest telling us:

```
disco/sagas/discovery.js:52
Discovery Sagas > fetches landing page addons from the API
```

instead of:

```
disco/sagas/discovery.js:52
disco/sagas/discovery > fetches landing page addons from the API
```